config.json

```diff
@@ -39,6 +39,11 @@
       "name": "ixod",
       "build": "v2",
       "version": "v0.18.1"
+    },
+    {
+      "name": "multisig",
+      "build": "v0.3.0",
```

Comment:
It's not a good idea to match the build version with the binary version. The build version's purpose is to differentiate among different GitHub Action builds of the binary. Multiple build versions can point to the same binary version.

The binary version is unique to the binary.

If you find a bug in the GitHub actions now and you need to rebuild the `multisig v0.3.0` binary, what will the build version be? `v0.3.1`? (You're not supposed to remove tags from the repo regularly.)

I would strongly suggest a `v1` build for multisig with the binary `v0.3.0` packaged.

Reply:
Thanks for this, I agree. It is now updated to v1